nightly-benchmark/run-dgx-benchmark.sh

```diff
@@ -19,6 +19,13 @@ ENV="$TODAY-nightly-0.17"
 conda activate $ENV
 which python
 
+echo "Cythonize Distributed"
+pushd "${CONDA_PREFIX}/lib/python3.8/site-packages/"
+cythonize -i \
+	"distributed/protocol/serialize.py"
```

Comment:
This already has some bits for Cython (though not everywhere). So went ahead and Cythonized it. Though if it causes issues for some reason, we can skip this as it is less critical atm.